webapp/templates/jury/submission.html.twig

```diff
@@ -123,13 +124,15 @@
                 {% else %}
                     {{ submission.problem.name }}
                 {% endif %}
+                {{ submission.problem | entityIdBadge('p') }}
             </a>
         </span>
 
         <span>
             <i class="fas fa-comments" title="Language:"></i>
             <a href="{{ path('jury_language', {'langId': submission.language.langid}) }}">
                 {{ submission.language.name }}
+                {{ submission.language | entityIdBadge }}
```

Comment:
Maybe for consistency sake we should prefix the language id too

Reply:
We never did this since language id’s are strings. lpy3 looks weird imho

Reply:
I know, that is why it is a very very minor nit.